### DISCUSSION TITLE
Bulk Export to download data, list of aliases

### DISCUSSION THREAD

**Discussion by ccorneli** [anonaddy/anonaddy#162] [Feature Requests]
I'm migrating to AnonAddy from another similar service, and one thing I find helpful from them is the ability to export all my aliases to CSV file.  This helps me work through a list of historical email aliases, and identify important ones which need to be recreated and redirected to AnonAddy.  It would also help with concerns about committing to AnonAddy, since a bulk export would also allow me to periodically save a copy, in case you suddenly disappear one day and I need to figure out what needs to be recovered.

As a bonus it would help to include info such as description, created date; last used, email count, etc... 

(I saw there was a related request about bulk export/import to help recreate aliases in bulk, but that sounded like a slightly different request)

**Comment by willbrowningme** [comment]
It is already possible to export data for all of your aliases by going to the bottom of the [settings](https://app.anonaddy.com/settings) page and clicking "export alias data".

The import aliases feature will only be available for custom domains when that is added.

**Reply by ccorneli** [reply]
Oh.. apologies.. I looked for it but somehow missed it.  And it is exactly what I had hoped.
